### PR TITLE
Add database prefix to status report

### PR DESCRIPTION
**Pull request by corsonr**
Add database prefix to the status report. Useful to troubleshoot issues such as missing links in downloadable files URLs missing as described here: https://docs.woothemes.com/document/completed-order-email-doesnt-contain-download-links/

### CODE REVIEW

**Comment by mikejolley** [comment]
@corsonr Needs an esc_html on the prefix output, and the description could do with a reword.

> We recommend using a prefix having less than 20 characters.

to:

> We recommend using a prefix with less than 20 characters.

That could even be output only if DB prefix > 20 as a warning.

**Comment by mikejolley** [comment]
To be clear I meant show the prefix, but show an error if its >20, not hide the entire column.

**Comment by corsonr** [comment]
Added error/success message:

File: includes/admin/views/html-admin-page-status-report.php [hunk 1]
```diff
@@ -346,13 +346,18 @@
...
 			<td><?php echo esc_html( get_option( 'woocommerce_db_version' ) ); ?></td>
 		</tr>
-		<?php if( strlen( esc_html( $wpdb->prefix ) ) > 20 ) { ?>
 		<tr>
 			<td data-export-label="WC Database Prefix"><?php _e( 'Database Prefix', 'woocommerce' ); ?></td>
-			<td class="help"><?php echo wc_help_tip( __( 'We recommend using a prefix with less than 20 characters.', 'woocommerce' ) ); ?></td>
-			<td><?php echo esc_html( $wpdb->prefix ) ?></td>
+			<td class="help">&nbsp;</td>
+			<td><?php 
+				if ( strlen( esc_html( $wpdb->prefix ) ) > 20 ) {
```
Comment (claudiosmweb) [inline includes/admin/views/html-admin-page-status-report.php]: Don't need to `esc_html()` here.

Comment (corsonr) [inline includes/admin/views/html-admin-page-status-report.php]: @claudiosmweb do you mean in `strlen()`?

Comment (claudiosmweb) [inline includes/admin/views/html-admin-page-status-report.php]: Yes, only `strlen( $wpdb->prefix ) > 20` is enough.

If the opposite may display the message to incorrect results, like:

```
wp> strlen( '<db_' )
int(4)
wp> strlen( esc_html( '<db_' ) )
int(7)
```

**Comment by shivapoudel** [comment]
@corsonr Could you wrap database prefix shown in warning with `<code>` tag.
capture

CC @claudiosmweb

**Pull request merged**